DependencyInjection/Configuration.php

```diff
@@ -125,6 +125,14 @@ public function getConfigTreeBuilder()
                             ->defaultTrue()
                             ->info('Enable automatic registration of annotations with JMSDiExtraBundle')
                         ->end()
+                        ->scalarNode('role_admin')
+                            ->defaultValue('ROLE_SONATA_ADMIN')
+                            ->cannotBeEmpty()
```

Comment:
Sure, I'm at work now but I'll look at it this evening. What about moving the configuration under the security key?